contracts/modules/royalty/RoyaltyModule.sol

```diff
@@ -537,6 +548,23 @@ contract RoyaltyModule is IRoyaltyModule, VaultController, ReentrancyGuardUpgrad
         _getRoyaltyModuleStorage().accumulatedRoyaltyPolicies[ipId].add(royaltyPolicy);
     }
 
+    /// @notice Transfers the royalty tokens to the royalty policy
+    /// @param parentIpId The parent IP asset
+    /// @param royaltyPolicy The address of the royalty policy
+    /// @param licensePercent The license percentage
+    /// @param ipRoyaltyVault The address of the ipRoyaltyVault
+    /// @return rtsRequiredToLink The required royalty tokens to link
+    function _transferRoyaltyTokensToPolicy(
+        address parentIpId,
+        address royaltyPolicy,
+        uint32 licensePercent,
+        address ipRoyaltyVault
+    ) internal returns (uint32) {
+        uint32 rtsRequiredToLink = IRoyaltyPolicy(royaltyPolicy).getPolicyRtsRequiredToLink(parentIpId, licensePercent);
+        IERC20(ipRoyaltyVault).safeTransfer(royaltyPolicy, rtsRequiredToLink);
+        return rtsRequiredToLink;
```

Comment:
if `rtsRequiredToLink` == 0, does not need to call Vault's `saftTransfer()` to safe gas.

Reply:
addressed in https://github.com/storyprotocol/protocol-core-v1/pull/307/commits/e050cb254c00caaa7978c2491ba72296d2368c67